stripe/src/main/java/com/stripe/android/view/PaymentAuthWebViewActivityViewModel.kt

```diff
@@ -45,7 +45,7 @@ internal class PaymentAuthWebViewActivityViewModel(
         resultData.value = Intent().putExtras(
             paymentResult.copy(
                 flowOutcome = StripeIntentResult.Outcome.CANCELED,
-                shouldCancelSource = true
+                shouldCancelSource = args.shouldCancelSource
```

Comment:
Should we maybe rename this method?

Reply:
Rename which method?

Reply:
`PaymentAuthWebViewActivityViewModel.cancelIntentSource`. With this change, it doesn't always cancel the source. Could be `createCancelationResult` or something along those lines?

On a related note: is there any reason this method is using live data? Can't we just... return the Intent? Seems unnecessarily complicated

Reply:
Yeah this likely doesn't need to be a `LiveData`, so we can change it. Feel free to rename as well.

Reply:
done